### DISCUSSION TITLE
Add ability to set progression types for cheevos

### DISCUSSION THREAD

**Discussion by UnliquidSnake** [RetroAchievements/RAIntegration#1044] [Ideas]
Make it possible to set progression for achievements inside the emulator.
Currently how I imagine it would look is:

- in the assets list, have a new Type column between Points and State;
- in the achievement editor, in between the Points dropdown and the code, put another dropdown which you can set to Progression, Win Condition or None. It would also count as a change to the achievement for the purposes of tracking that sort of thing.

**Comment by redwizard42** [comment]
Please also provide us the ability to select multiple in the list and "Set Selected as <type>"

**Comment by Jamiras** [comment]
Progression type support was added by #1052. Multi-edit is not yet supported.
